src/Tempest/Support/src/ArrayHelper.php

```diff
@@ -286,11 +564,21 @@ public function unwrap(): self
         return new self($array);
     }
 
+    public function dump(mixed ...$dumps): self
+    {
+        dump($this->array, ...$dumps); // @phpstan-ignore-line
```

Comment:
You should use `lw` instead of `dump` here, `lw` integrates with Tempest's logger as well: https://tempestphp.com/docs/framework/logging/